test/test_jsonrpcbase.py

```diff
@@ -1,40 +1,37 @@
 """
 jsonrpcbase tests
 """
-
 import jsonrpcbase
-from nose.tools import assert_equal, assert_not_equal
-import six
 
 s = jsonrpcbase.JSONRPCService()
 
 
-def subtract(a, b):
-    return a - b
+def subtract(params, meta):
```

Comment:
All methods now have these uniform parameters, where `params` is the deserialized json data